app/views/outcomes-report/PageOutcomesReport.vue

```diff
@@ -438,9 +438,9 @@ main#page-outcomes-report
           label.edit-label.editing-only(v-if="editing" for="startDate") &nbsp; (edit)
 
     .org-results(v-if="org && !loading")
-      outcomes-report-result-component(:org="org" :other-org="otherOrg" v-bind:editing="editing" :showLicense="showLicense" :showLicenseSummary="showLicenseSummary && kind !== 'student'" :showOther="showOther")
+      outcomes-report-result-component(:org="org" :other-org="otherOrg" v-bind:editing="editing" :showLicense="showLicense" :showLicenseSummary="showLicenseSummary && kind !== 'student'" :showOther="showOther" :org-id="orgIdOrSlug" :org-kind="kind")
```

Comment:
why do we add `org-id` and `org-kind` ? i don't see usage. also, `org` should contains them ?